docs/en/observability/slo-troubleshoot.asciidoc

```diff
@@ -0,0 +1,259 @@
+
+* SLO configuration
+* Rollup transform configuration
+* Summary transform configuration
+* Rollup ingest pipeline
+* Summary ingest pipeline
+* Temporary document
+* Rollup transform query composite
+* Summary transform query composite
+
+These resources are very useful for tasks such as trying out the queries performed by the transforms and checking the IDs of all associated resources. The view also includes direct links to transforms and ingest pipelines sections in {kib}.
+
+[discrete]
+[[slo-troubleshoot-reset]]
+=== Reset SLO
+
+Resetting an SLO forces the deletion of all SLI data, summary data, and transforms, and then reinstalls and processes the data. Essentially, it recreates the SLO as if it had been deleted and re-created by the user.
+
+[NOTE]
+====
+While resetting an SLO can help resolve certain issues, it may not always address the root cause of errors. Most errors related to transforms typically arise from improperly structured source data, such as unparsable timestamps, which prevent the transform from progressing. Additionally, incorrect formatted SLO queries, and consequently transform queries, can also lead to failures.
+
+Before resetting the SLO, verify that the source data and queries are correctly formatted and validated. Resetting should only be used as a last resort when all other troubleshooting steps have been exhausted.
+====
+
+Follow these steps to reset an SLO:
+
+. Find **SLOs** in the main menu or use the {kibana-ref}/introduction.html#kibana-navigation-search[global search field].
+. Click on the SLO to reset.
+. Select *Actions* → *Reset*.
+
+Alternatively you can use {kib} API for the reset action:
+
+[source,console]
+----
+POST kbn:/api/observability/slos/{sloId}∫/_reset
+----
+
+Where `sloId` can be obtained from the <<slo-troubleshoot-inspect>> action.
+
+[discrete]
+[[slo-troubleshoot-beta]]
+== Upgrade from beta to GA
+
+Starting in version 8.12.0, SLOs are generally available (GA).
+If you're upgrading from a beta version of SLOs (available in 8.11.0 and earlier),
+you must migrate your SLO definitions to a new format. Otherwise SLOs won't show up.
+
+[%collapsible]
+.Migrate your SLO definitions
+====
+To migrate your SLO definitions, open the SLO overview.
+A banner will display the number of outdated SLOs detected.
+For each outdated SLO, click **Reset**. If you no longer need the SLO, select **Delete**.
+
+If you have a large number of SLO definitions, it is possible to automate this process.
+To do this, you'll need to use two Elastic APIs:
+
+* https://github.com/elastic/kibana/blob/9cb830fe9a021cda1d091effbe3e0cd300220969/x-pack/plugins/observability/docs/openapi/slo/bundled.yaml#L453-L514[SLO Definitions Find API] (`/api/observability/slos/_definitions`)
+* https://github.com/elastic/kibana/blob/9cb830fe9a021cda1d091effbe3e0cd300220969/x-pack/plugins/observability/docs/openapi/slo/bundled.yaml#L368-L410[SLO Reset API] (`/api/observability/slos/${id}/_reset`)
```

Comment:
@lucabelluccini , this is interesting! I have updated (fixed) the second link (SLO RESET) because that one exist in our API docs: https://www.elastic.co/docs/api/doc/kibana/v8/operation/operation-resetsloop

But in the first one we have two problems:
- First of all there's a bug in the text and the API is not `api/observability/slos/_definition`, but `internal/observability/slos/_definitions` (based on the link).
- Secondly that API call is not in the public API docs.
- Thirdly (:-D) that link doesn't work for the `main` branch, so it all looks weird.

What would you suggest? should we check with devs?

Reply:
@kdelemme : would you be able to share anything on this topic? This isn't really related with the goal of this PR, but it's something that we have found out.